<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>PicoHarp</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="picoharp_tacq_doubleSpinBox"/>
      </item>
      <item row="9" column="5">
       <widget class="QDoubleSpinBox" name="ch0_doubleSpinBox"/>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Channel 0</string>
        </property>
       </widget>
      </item>
      <item row="10" column="5">
       <widget class="QDoubleSpinBox" name="ch1_doubleSpinBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Binning</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Channel 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Integration Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>histogram channels</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QDoubleSpinBox" name="histogram_channels_doubleSpinBox"/>
      </item>
      <item row="1" column="5">
       <widget class="QComboBox" name="Binning_comboBox"/>
      </item>
      <item row="3" column="5">
       <widget class="QCheckBox" name="use_calc_hist_chans_checkBox">
        <property name="text">
         <string>auto histogram channels</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
